packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -315,6 +316,28 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
           }
         });
       }
+    } else if (
+      appStoreMetadata[credential.app?.slug as keyof typeof appStoreMetadata].extendsFeature === "EventType"
```

Comment:
I don't think this is the full fix, as the error was about 'cannot set extendsFeature on undefined'.

- I think it was possible that appStoreMetadata[credential.app?.slug as keyof typeof appStoreMetadata] can be undefined and in those situations can still crash. may need:

appStoreMetadata[credential.app?.slug as keyof typeof appStoreMetadata]?.extendsFeature === "EventType"